iOS Example Frame SPM/.gitignore

```diff
@@ -0,0 +1,2 @@
+# SwiftLint Remote Config Cache
+.swiftlint/RemoteConfigCache
```

Comment:
Do we want to have this under source control? Don't see the file being committed anyway so not sure what benefit it adds?

Reply:
It was auto added. cache should be ignored.